lib/modules/platform/bitbucket/schema.ts

```diff
@@ -30,7 +30,9 @@ export const RepoInfo = z
       name: z.string(),
     }),
     has_issues: z.boolean().catch(() => {
-      logger.once.warn('Bitbucket: "has_issues" field missing from repo info');
+      logger.once.debug(
+        'Bitbucket: "has_issues" field missing from repo info. Issues are not available for this repository.',
+      );
```

Comment:
```suggestion
```

Let's actually drop this altogether, as we expect it to happen a lot from now on